userComponent/src/main/kotlin/com/gravatar/app/usercomponent/di/HttpClientModule.kt

```diff
@@ -0,0 +1,22 @@
+package com.gravatar.app.usercomponent.di
+
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.okhttp.OkHttp
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.serialization.kotlinx.json.json
+import kotlinx.serialization.json.Json
+import org.koin.dsl.module
+
+internal val httpClientModule = module {
+    single {
+        HttpClient(OkHttp) {
+            install(ContentNegotiation) {
+                json(
+                    Json {
+                        ignoreUnknownKeys = true
+                    }
+                )
+            }
+        }
+    }
```

Comment:
We can configure more stuff here, so we don't have to repeat it in each request, but let's do that as we need it. This is also the place to configure stuff like network loggers if we need to in the future.